app/project_type/git.py

```diff
@@ -19,6 +19,19 @@ class Git(ProjectType):
     DIRECTORY_PERMISSIONS = 0o700
 
     @staticmethod
+    def _replace_invalid_path_characters_from_netloc(netloc):
+        """
+        Replace the following characters in netloc.
+        ':' -> '-' (':' is not a valid character in path on Windows)
+
+        :param netloc: Network location
+        :type netloc: str
+        :return: Path after replacement
+        :rtype: str
+        """
+        return netloc.replace(':', '-')
```

Comment:
I think we might need to replace dashes also. The comment in the git history of the `remove_invalid_path_characters` method says "The PHP Intl extension doesn't appear to work if the repo is in a directory with a dash in the path."

And we should document that in a code comment as well. :)